Requisitos

```diff
@@ -0,0 +1,66 @@
+## Requisitos do Trabalho
+
+### 1. Identificação do Projeto
+
+**Nome do projeto:** Saúde Goiás
+
+### 2. Descrição do Projeto
+
+**Objetivo do projeto:** Facilitar o acesso à informação sobre saúde pública no estado de Goiás.
+
+**Visão geral do projeto:** Uma aplicação web que apresenta dados de saúde da região de Goiás de forma gráfica e intuitiva.
+
+**Funcionalidades principais:**
+
+* Visualização de dados de saúde em gráficos e tabelas.
+* Filtros para refinar a pesquisa por município, indicador de saúde, período, etc.
+* Interface amigável e acessível para diferentes públicos.
+
+**Público-alvo:**
+
+* Estudiosos e pesquisadores da área da saúde.
+* Gestores públicos e tomadores de decisão.
+* População em geral interessada em saúde pública.
+
+### 3. Requisitos Funcionais
+
+**Funcionalidade 1: Tabela de dados**
+
+**Descrição da funcionalidade:** Uma tabela que apresenta os dados da região de forma organizada e responsiva.
+
+**Requisitos:**
+
+* A tabela deve ser organizada por colunas, com títulos claros e concisos.
+* A tabela deve ser responsiva, adaptando-se automaticamente a diferentes tamanhos de tela.
+* Os usuários devem poder filtrar os dados por município, indicador de saúde, período, etc.
+
+
+**Exemplo de tabela:**
+
+| Município | Indicador de saúde | Período | Valor | Fonte dos dados |
+|---|---|---|---|---|
+| Goiânia | Esperança de vida ao nascer | 2020 | 77,5 anos | IBGE |
+| Aparecida de Goiânia | Mortalidade infantil | 2020 | 12,5 por mil nascidos vivos | SES-GO |
+| Anápolis | Taxa de internação por doenças respiratórias | 2020 | 100 por 100 mil habitantes | DATASUS |
+
+**Observações:**
+
+* A tabela acima é apenas um exemplo. Os dados reais podem variar.
+* A tabela deve ser customizada de acordo com as necessidades do projeto.
+
+**Funcionalidades adicionais (podem ser adicionadas no decorrer do projeto):**
+
+* Gráficos e mapas interativos para visualizar os dados de forma mais intuitiva.
+* Painéis de controle com indicadores chave de desempenho (KPIs) da saúde pública em Goiás.
+* Blog com notícias e informações sobre saúde pública.
+* Fórum para discussão de temas relacionados à saúde pública.
```

Comment:
Um fórum ou espaço para comentários pode ser muito útil.
O site utilizará gráficos e mapas interativos (decidir qual gráficos utilizaremos).
(KPIs serão adotadas)
Blog somente se for adicionado com links de terceiros com informações úteis, nós mesmos integrarmos  no site não condiz com objetivo principal.